examples/chart/teleport-cluster/templates/NOTES.txt

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.highAvailability.certManager.enabled }}
```

Comment:
where do these notes show up?

Reply:
They appear on the command line when you run `helm install`. You can also show them with `helm get notes <release-name>`.